File: Trees/Tree Representation.c
#include<stdio.h>
#include<stdlib.h>

struct Node
{
    int data;
    struct Node *left;
    struct Node *right;
};


struct Node* create(int data)
{
    struct Node * root = (struct Node*)malloc(sizeof(struct Node));
    root->left = NULL;
    root->right = NULL;
    return root;
}

int main()
{
    struct Node* root = create(1);
}

File: Trees/Tree Representation.java
import java.util.*;

class Node
{
    int data;
    Node left, right;
    
    Node(int val)
    {
        data = val;
        left = null;
        right = null;
    }

}

class Main
{
    public static void main(String args[])
    {
        Node root = new Node(0);
        root.left = new Node(1);
        root.right = new Node(2);
    }
}